docker-scripts/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:18.04
+
+#Build in this directory with sudo docker build . or docker build . (depending on your OS)
+ENV GOROOT=/usr/local/go
+ENV GOPATH=/opt/sensu
+ENV PATH=$GOPATH:/bin:$GOROOT/bin:/opt/sensu/bin/:$PATH
+
+WORKDIR /opt/sensu
+COPY ./sensu-entrypoint.sh /opt/sensu/bin/
+
+RUN apt update &&\
+    apt -y upgrade &&\
+    apt -y install git wget
+
+#Download and install Go 1.13.3 for Go modules.
+RUN wget https://dl.google.com/go/go1.13.3.linux-amd64.tar.gz && tar -xvf go1.13.3.linux-amd64.tar.gz &&\
+    mv go /usr/local/ &&\
+    rm go1.13.3.linux-amd64.tar.gz
+
+#Build OSS only sensu features, enterprise features are built in the Sensu supported images:
+#docker pull sensu/sensu (Alpine based container) or 
+#docker pull sensu/sensu-rhel (Red Hat based container)
+RUN git clone --depth=1 https://github.com/sensu/sensu-go.git &&\ 
+    cd sensu-go &&\
+    go build -o ../bin/sensu-agent ./cmd/sensu-agent &&\
+    go build -o ../bin/sensu-backend ./cmd/sensu-backend &&\
+    go build -o ../bin/sensuctl ./cmd/sensuctl &&\
+    rm -rf /opt/sensu/sensu-go
+
+#State directory lives outside container for persistent data.
+VOLUME [/var/lib/sensu]
+#Port 3000 for Sensu Go UI, 8080 for Sensu API, and port 8081 for sensu-agent websocket API communication.  
+EXPOSE 3000 8080 8081
+
+#Use Docker logs to see stdout of sensu-backend (ie. docker logs --details -f  `docker ps -q` )
+CMD ["/opt/sensu/bin/sensu-backend","start"]
```

Comment:
I am thinking we likely want a more standard `/entrypoint.sh` or something along the lines of that so that people can choose depending on their needs if they need to run `/opt/sensu/bin/sensu-backend init` prior to starting it to avoid seeding with default credentials.

Reply:
There is the docker-scripts/sensu-entrypoint.sh, we could use that, or do you suggest another script?

Reply:
that one seems reasonable from a quick look.

Reply:
Ok, I have updated the README.md and Dockerfile for this change.  Should be good to go.